### ISSUE TITLE
 error TS2503: Cannot find namespace 'angular'.

### ISSUE THREAD

**Issue by 3du4** [DanielSchuech/ngAdapter#11]
I am getting this error after executing tsc.    

node_modules/ngAdapter/build/ngAdapter.d.ts(11,25): error TS2503: Cannot find namespace 'angular'.    

My typings file looks like this:

`{`
  `"globalDependencies": {`
    `"core-js": "registry:dt/core-js#0.0.0+20160725163759",`
    `"jasmine": "registry:dt/jasmine#2.2.0+20160621224255",`
    `"node": "registry:dt/node#6.0.0+20160909174046"`
  `}`
`}`

**Comment by DanielSchuech** [comment]
Add the typings for AngularJS.

add the following to globalDependencies:

```
"angular": "github:DefinitelyTyped/DefinitelyTyped/angularjs/angular.d.ts#5256422ee72d2b1ab41bbb72eb47754e202b6bb8"
```

and execute typings install again

You can also use the AngularJS typings from the npm registry.

**Comment by 3du4** [comment]
@DanielSchuech, Thank you.

I added that line and executed typings install again. Then I got this error:
typings/globals/angular/index.d.ts(979,43): error TS2304: Cannot find name 'JQuery'.

I also updated core-js, jode and jasmine to the latest versions including their tags.
Now my typings.js looks like this

`{`
  `"globalDependencies": {`
    `"core-js": "registry:dt/core-js#0.0.0+20160914114559",`
    `"jasmine": "registry:dt/jasmine#2.2.0+20160621224255",`
    `"node": "registry:dt/node#6.0.0+20160928143418",`
 `"angular":"github:DefinitelyTyped/DefinitelyTyped/angularjs/angular.d.ts#5256422ee72d2b1ab41bbb72eb47754e202b6bb8"`
  `}`
`}`
